index.bs

```diff
@@ -193,17 +199,15 @@ The <dfn method for="StorageBucketManager">delete(|name|)</dfn> method steps are
 
 1. If |shelf| is failure, then return [=a promise rejected with=] a {{TypeError}}.
 
-1. Let |p| be [=a new promise=].
+1. If the result of [=validate a bucket name=] with |name| is failure, then return [=a promise rejected with=] an {{InvalidCharacterError}}.
```

Comment:
nit, maybe:

```
1. Let p be a new promise
2. If the result of validate is failure, then reject p with error
3. Otherwise, enqueue these steps: ...
4. Return p.
```

Reply:
Updated!